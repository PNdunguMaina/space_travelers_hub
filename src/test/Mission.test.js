import React from 'react';
import Renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Missions from '../components/missions/MissionsContainer';

test('renders missions correctly', () => {
  const tree = Renderer.create(
    <React.StrictMode>
      <Provider store={store}>
        <Router>
          <Missions />
        </Router>
      </Provider>
    </React.StrictMode>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
