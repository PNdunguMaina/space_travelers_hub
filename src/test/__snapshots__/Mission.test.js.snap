// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders missions correctly 1`] = `
<div>
  <table
    className="table table-striped table-bordered"
  >
    <tHead>
      <tr>
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
      </tr>
    </tHead>
    <tBody />
  </table>
</div>
`;
